### ISSUE TITLE
Weather providers with Let’s encrypt certificate will no longer work on Android < 7.0 by 2024-09-30

### ISSUE THREAD

**Issue by papjul** [breezy-weather/breezy-weather#158]
### Describe your suggested feature

As reported on Matrix server, see https://letsencrypt.org/2023/07/10/cross-sign-expiration.html

We need to take action to trust Let’s encrypt on Android < 7.1.
This should be done before 2024-02-08 when cross-sign will stop happening.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to **[latest version](https://github.com/breezy-weather/breezy-weather/releases/latest)**.

**Comment by papjul** [comment]
Instructions: https://developer.android.com/privacy-and-security/security-config#TrustingAdditionalCas
According to Stack Overflow, this may be more difficult to implement on SDK < 24:
https://stackoverflow.com/questions/65304273/handling-lets-encrypt-new-certificates-in-webview-for-android-m-and-older

Certificates: https://letsencrypt.org/certificates/
ISRG Root X1, do we also need ISRG Root X2?

**Comment by papjul** [comment]
Support was added for versions 7.0-7.1.0 in 64963d6d19804618d4c1e760781485821c15519a

For < 7.0, let’s check https://github.com/square/okhttp/blob/okhttp_4.10.x/samples/guide/src/main/java/okhttp3/recipes/CustomTrust.java
